core/src/main/scala/slog/LogBuilder.scala

```diff
@@ -0,0 +1,7 @@
+package slog
+
+trait LogBuilder[F[_]] {
+  def msg(msg: String): F[Unit]
+  def msg(ex: Throwable)(msg: String): F[Unit]
+  def context[T: StructureEncoder](key: String, value: => T): LogBuilder[F]
```

Comment:
I don' like the separation between creating a context and logging things. `context` and `use` do kinda the same thing.

Reply:
The point is to allow multiple context to be used.

Reply:
I don't understand... you can call `context` multiple times, right?

Reply:
Ahh, there was a misunderstanding. I kind of didn't like to use `loggingContext.context`.

Reply:
I would like those methods which have signature like `(key: String, value: T)` have the same name. Could it be `with`?
I don't personally care much about which specific name is chosen...

Reply:
`with` is a keyword, try something else:)